find_primes.py

```diff
@@ -1 +1,29 @@
+#num1 is the original number, num2 is the prime number, num3 is all of the primes,
+num1 = int(input("Enter any positive integer: "))
+num2 = num1 + 1
+num3 = []
+count = 0
+
+def find_prime(a):
+  '''find_prime(a) -> yes or no
+  Prints whether a is a prime'''
+  import math
+  limit = int((math.sqrt(a)) + 1)
+  for i in range(2, limit):
+    value1 = a / i
+    value2 = int(a / i)
+    if value1 == value2:
+      return "no"
+  return "yes"
+
+while count != 5:
```

Comment:
Let’s make the number of primes to be found something that the user can input as well, rather than having it always be 5.